### PR TITLE
fix: RSS feed

### LINKED ISSUE #391: RSS feed isn't correct
hi Josh! I tried to add your blog's RSS feed to my feed reader and found some issues to address:

https://blog.joshuakgoldberg.com/rss.xml _does_ exist, but has some problems:

1. it doesn't have recent content
2. the encoding and/or formatting doesn't seem correct(?)

	- image
	- (latest Firefox)
	- image
	- (notice the tell-tale `&gt;`-style encoding)

I'm not terribly familiar with Gatsby but it seems like there's a pretty reasonable out-of-the-box solution: https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/

I might see if I can take a stab at a PR to fix this. But regardless, wanted to let you know. :)

P.S. Your learningtypescript.com book looks excellent, nice work!

### PR DESCRIPTION
**Pull request by Pinjasaur**
Closes #391.

Took a stab at fixing the borked RSS feed. Essentially just followed the documentation here: https://www.gatsbyjs.com/plugins/gatsby-plugin-feed/

A few other noteworthy changes:

- `.nvmrc` file, set to the the latest LTS release
- note in the README re: Yarn 1.x (tripped me up briefly)
- `yarn build:serve` script which does a `gatsby build && gatsby serve`

I didn't do substantial verification whatsoever. Essentially ran the `yarn build:serve` command I created and had a peek at the local `/rss.xml` and it looked fine, and at the very least appeared to have the most recent blog post.

FWIW, I'm not sure if you're "into" consuming ✨ things ✨ via RSS & friends, but if so, I'd definitely recommend dogfooding your own feed(s) just as a bit of a sanity check.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18.17.0
```
Comment (Pinjasaur) [inline .nvmrc:1]: If you aren't interested in `nvm` as tooling, feel free to toss this or I can.

File: gatsby-config.js [hunk 2]
```diff
@@ -76,5 +76,59 @@ module.exports = {
...
+                                        frontmatter {
+                                            title
+                                            date
+                                        }
+                                    }
+                                }
+                            }
+                            `,
+                            output: "/rss.xml",
+                            title: "The RSS feed for Josh Goldberg's personal blog.",
```
Comment (Pinjasaur) [inline gatsby-config.js:128]: You may want to change this.

**Comment by Pinjasaur** [comment]
Hm. That's weird, Vercel seems unhappy? I don't believe I'm able to take a peek at what's going on from my side of things, unfortunately.

File: .nvmrc (hunk 1 above)
Comment (JoshuaKGoldberg) [reply .nvmrc:1]: I like nvm :)

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Hooray!
 
 ## 🚀 Development
 
-After installing [Node >=14](https://nodejs.org/en/download), set up the repository locally with:
+After installing [Node >=14](https://nodejs.org/en/download) and [Yarn 1.x](https://classic.yarnpkg.com/en/docs/install), set up the repository locally with:
```
Comment (JoshuaKGoldberg) [inline README.md:8]: Heh, yeah, the Yarn 1->berry/next change has been a minor pain in my side for a while. One of the reasons I've started using pnpm in newer repos.

**Review by JoshuaKGoldberg — APPROVED** [review]
❤️‍🔥 this is wonderful - a task I've been procrasting looking into for over a year now, out of dread for dealing with Gatsby things. Thanks!

I'll make sure the new site also has an equivalently working rss.xml. And I'll better document in this repo that it's going away soon.

**Comment by JoshuaKGoldberg** [comment]
> Vercel seems unhappy?

I'd set the Node version to 16 years ago and hadn't updated it since. I bumped it to 18 and redeployed. ✅

File: gatsby-config.js (hunk 2 above)
Comment (JoshuaKGoldberg) [reply gatsby-config.js:128]: I'll think on it :) this seems reasonable

**Pull request merged**